Assets/MRTK/StandardAssets/Shaders/MixedRealityStandard.shader

```diff
@@ -316,6 +316,8 @@ Shader "Mixed Reality Toolkit/Standard"
 #endif
             };
 
+            CBUFFER_START(UnityPerMaterial)
+
 #if defined(_INSTANCED_COLOR)
             UNITY_INSTANCING_BUFFER_START(Props)
```

Comment:
This nested buffer within the per material constant buffer might not be supported? I'm not 100% sure, but would be surprised if it is.